packages/bpk-component-modal/src/BpkModal.tsx

```diff
@@ -71,12 +74,28 @@ const BpkModal = ({
   modalStyle = MODAL_STYLING.default,
   onClose = () => null,
   padded = true,
-  renderTarget = null,
+  renderTarget = () => null,
```

Comment:
looking at the TS error in CI, it seems like we'll have to do the same for the datepicker